java-test-samples/step-functions-local/README.md

```diff
@@ -0,0 +1,28 @@
+# Step Functions Local Testing using JUnit and Spock
+This project contains information about the [sample test application](https://github.com/aws-samples/aws-stepfunctions-examples/tree/main/sam/demo-local-testing-using-java) that highlights use of JUnit and Spock test framework to test Step functions locally.
+
+## Workflow Under Test
+The system under test in this pattern is a Step Functions Workflow with multiple service integrations to AWS Lambda, Amazon Comprehend, and Amazon DynamoDB.
+
+![Step Functions Workflow](images/stepfunctions_local_test.png)
+
+
+## Goal
+This pattern demonstrates how to use mock service integrations to test Step Functions Workflows when using Step Functions Local. 
+The pattern enables developers to define sample outputs from AWS service integrations. 
+The mocked responses can be combined into test case scenarios to validate workflow control and data flow definitions. 
+This pattern is intended to perform rapid functional testing without affecting cloud resources. 
+Testing is conducted in a self-contained local environment. 
+This pattern enables rapid development and testing of a Step Functions Workflow that makes calls to other AWS cloud services using service integrations. 
+This pattern eliminates the need to perform a build and deploy of the Step Functions Workflow to the cloud between modifications of the workflow definition. 
```

Comment:
Maybe bring lines 16 or 17 higher up in this list - helps developers understand the benefits of this approach vs the implementation details, which could come later in the list

Reply:
Done